tests/integration/__init__.py

```diff
@@ -819,6 +823,38 @@ def setup_keyspace(ipformat=None, wait=True, protocol_version=None, port=9042):
         cluster.shutdown()
 
 
+def is_scylla_enterprise(version: Version) -> bool:
+    return version > Version('2000.1.1')
+
+
+def xfail_scylla_version_lt(reason, oss_scylla_version, ent_scylla_version, *args, **kwargs):
+    """
+    It is used to mark tests that are going to fail on certain scylla versions.
+    :param reason: message to fail test with
+    :param oss_scylla_version: str, oss version from which test supposed to succeed
+    :param ent_scylla_version: str, enterprise version from which test supposed to succeed. It should end with `.1.1`
+    """
+    if not reason.startswith("scylladb/scylladb#"):
+        raise ValueError('reason should start with scylladb/scylladb#<issue-id> to reference issue in scylla repo')
+
+    if not isinstance(ent_scylla_version, str):
+        raise ValueError('ent_scylla_version should be a str')
+
+    if not ent_scylla_version.endswith("1.1"):
+        raise ValueError('ent_scylla_version should end with "1.1"')
+
```

Comment:
If some bug is fixed (or some feature added) in a version that does not end in 1.1, allowing us to enable some test, how do we use this decorator to describe this? It does require the version to end in 1.1.

Reply:
We don't, when it happened we will have to go with code it was there before in https://github.com/scylladb/python-driver/pull/363 .